test/e2e/fpgaadmissionwebhook/fpgaadmissionwebhook.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fpgaadmissionwebhook
+
+import (
+	"os"
+	"time"
+
+	"github.com/intel/intel-device-plugins-for-kubernetes/test/e2e/utils"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+)
+
+const (
+	deployScript = "scripts/webhook-deploy.sh"
+)
+
+func init() {
+	ginkgo.Describe("FPGA Admission Webhook", describe)
+}
```

Comment:
nit: the shorter, canonical way of writing this is:
```
var _ = ginkgo.Describe("FPGA Admission Webhook", func() {
....
})
```

Just a matter of taste, of course.

Reply:
I prefer to have modules' entry points specified explicitly, but perhaps for ginkgo it makes sense to make an exception indeed to avoid the need of inventing names for the `describe()` functions.